.github/workflows/add-to-inbox.yml

```
@@ -0,0 +1,38 @@
+name: Add to Inbox 📥
+on:
+  issues:
+    types: [opened, reopened]
+
+jobs:
+  add-to-inbox:
+    if: ${{ github.repository == 'primer/behaviors' }}
+    runs-on: ubuntu-latest
+    env:
+      ISSUE_URL: ${{ github.event.issue.html_url }}
+      PROJECT_ID: 4503
+    steps:
+      - id: get-primer-access-token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ vars.PRIMER_ISSUE_TRIAGE_APP_ID }}
+          private-key: ${{ secrets.PRIMER_ISSUE_TRIAGE_APP_PRIVATE_KEY }}
+      - name: Add rails label to issue
+        run: |
+          gh issue edit $ISSUE_URL --add-label rails
+        env:
+          GH_TOKEN: ${{ steps.get-primer-access-token.outputs.token }}
+      - name: Add react label to issue
+        run: |
+          gh issue edit $ISSUE_URL --add-label react
+        env:
+          GH_TOKEN: ${{ steps.get-primer-access-token.outputs.token }}
```

Just wanted to share a couple of options to refactor if they'd be helpful!

I think `--add-label` supports multiple labels through a comma:

```suggestion
      - name: Add labels to issue
        run: |
          gh issue edit $ISSUE_URL --add-label 'rails,react'
        env:
          GH_TOKEN: ${{ steps.get-primer-access-token.outputs.token }}
```

Another idea was to just do the command one after the other if we want them separate:

```suggestion
      - name: Add rails label to issue
        run: |
          gh issue edit $ISSUE_URL --add-label rails
          gh issue edit $ISSUE_URL --add-label react
        env:
          GH_TOKEN: ${{ steps.get-primer-access-token.outputs.token }}
```

---

> I think `--add-label` supports multiple labels through a comma:

TIL! Thanks for simplifying. Looks like the merged code follows this suggestion: https://github.com/primer/behaviors/pull/239/commits/06cef459c90f3afbe841d003c5db3f710a08a0b4